Airports/EGSC/SMR/Labels.txt

```diff
@@ -6,6 +6,7 @@
 "L" N052.12.13.788 E000.10.06.870 standHold
 "J" N052.12.31.107 E000.10.46.853 standHold
```

Comment:
Just while we've got a pull request open for this, can I be _really_ annoying and ask you to move the J label above the L one so as to alphabetically order them! :D hehe

Reply:
@hsugden done :smile: